lib/main.dart

```diff
@@ -1,177 +1,537 @@
+        // It is responsive and change style according to the device.
+        body: BlocBuilder<TodoBloc, TodoState>(
+          builder: (context, state) {
+            // If the list is loaded
+            if (state is TodoListLoadedState) {
+              List<Item> items = state.items;
+
+              return SafeArea(
+                child: Column(
+                  children: [
+                    ResponsiveLayout(
+                      // On mobile
+                      mobileBody: TextField(
+                          key: textfieldKey,
+                          controller: TextEditingController(),
+                          keyboardType: TextInputType.none,
+                          onTap: () {
+                            Navigator.of(context)
+                                .push(navigateToNewTodoItemPage());
+                          },
+                          maxLines: 2,
+                          style: const TextStyle(fontSize: 20),
+                          decoration: const InputDecoration(
+                              border: OutlineInputBorder(
+                                  borderRadius: BorderRadius.zero),
+                              hintText: 'Capture more things on your mind...'),
+                          textAlignVertical: TextAlignVertical.top),
+
+                      // On tablet and up
+                      tabletBody: TextField(
+                          key: textfieldKey,
+                          controller: TextEditingController(),
+                          keyboardType: TextInputType.none,
+                          onTap: () {
+                            Navigator.of(context)
+                                .push(navigateToNewTodoItemPage());
+                          },
+                          maxLines: 2,
+                          style: const TextStyle(fontSize: 30),
+                          decoration: const InputDecoration(
+                              border: OutlineInputBorder(
+                                  borderRadius: BorderRadius.zero),
+                              hintText: 'Capture more things on your mind...'),
+                          textAlignVertical: TextAlignVertical.top),
+                    ),
 
-        // Menu drawer (`endDrawer` comes from the right of the screen)
-        endDrawer: SizedBox(
-            width: MediaQuery.of(context).size.width * 1.0,
-            child: const Drawer(child: DrawerMenu())));
+                    // List of items
+                    Expanded(
+                      child: ListView(
+                        padding: const EdgeInsets.only(top: 40),
+                        scrollDirection: Axis.vertical,
+                        shrinkWrap: true,
+                        children: [
+                          if (items.isNotEmpty) const Divider(height: 0),
+                          for (var i = 0; i < items.length; i++) ...[
+                            if (i > 0) const Divider(height: 0),
```

Comment:
```suggestion
                            if (i > 0) const Divider(height: 0),
```

maybe a typo? a divider with no height is the same as not having no divider iirc

Reply:
It's not a typo :) 

You can see it in this very much *basic* layout.

![image](https://github.com/dwyl/app/assets/17494745/9b9400a8-1ce0-4031-885b-114ea9347ca4)